CHANGES.md

```diff
@@ -1,5 +1,22 @@
 # Changes
 
+## edge-21.3.4
+
+This edge release is a release candidate for stable 2.10.1 and fixes
```

Comment:
Is it necessary to call this out as an RC for 2.10.1?  Given that we don't know exactly when 2.10.1 will be released or whether it will branch off of main or not, it might be better to not say this.

Reply:
Makes sense. Removed this